wsi_superpixel_guided_labeling/web_client/views/vue/components/ActiveLearning/ActiveLearningContainer.vue

```diff
@@ -95,8 +101,8 @@ export default Vue.extend({
         },
         changeLog: {
             handler() {
-                const change = this.changeLog[this.changeLog.length - 1];
-                this.backboneParent.saveLabelAnnotations([change.imageId]);
+                const change = this.changeLog.pop();
```

Comment:
@naglepuff It would be good to get your confirmation that we do not need to keep the change in the changeLog list after we've handled it here. By consuming the change we prevent triggering this watcher and re-applying the annotation layer on slider change (which makes rendering jumpy).